### ISSUE TITLE
bug: Event rule is created with bad values

### ISSUE THREAD

**Issue by russelltaga** [localstack/localstack#4394]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

When creating an event bus with a rule that triggers a Lambda function via [cdklocal](https://github.com/localstack/aws-cdk-local), the event rule is generated with some values that are incorrect.

https://github.com/coast-app/localstack-event-rule-bug#issue

### Expected Behavior

The rule should be generated with the correct values.

https://github.com/coast-app/localstack-event-rule-bug#expected-values

### How are you starting LocalStack?

With a docker-compose file

### Steps To Reproduce

Please see steps to reproduce here

https://github.com/coast-app/localstack-event-rule-bug#localstack-set-up

### Environment

```markdown
- OS: macOS 11.4
- LocalStack: Please see https://github.com/coast-app/localstack-event-rule-bug#localstack-set-up
```

### Anything else?

Created this github repo to make it easy to reproduce the problem and included details on workarounds

https://github.com/coast-app/localstack-event-rule-bug

**Comment by whummer** [comment]
Thanks for the details report @russelltaga ! The sample project was really helpful in replicating the issue end-to-end. This should be fixed in 4e15f778a1a4b4beebd0e25646277f6ae04a28ea - can you please pull the `latest` Docker image and give it another try? Please let us know if the problem persists. Thanks for your help!

**Comment by russelltaga** [comment]
Verified fix. Thank you so much for making that happen so quickly!
